WebExample/tests/textManipulation.spec.ts

```diff
@@ -0,0 +1,139 @@
+import {test, expect} from '@playwright/test';
+import type {Locator, Page} from '@playwright/test';
+import * as TEST_CONST from '../../testConstants';
+import {checkCursorPosition, setupInput} from './utils';
+
+const OPERATION_MODIFIER = process.platform === 'darwin' ? 'Meta' : 'Control';
+
+const pasteContent = async ({text, page, inputLocator}: {text: string; page: Page; inputLocator: Locator}) => {
+  await page.evaluate(async (pasteText) => navigator.clipboard.writeText(pasteText), text);
+  await inputLocator.focus();
+  await inputLocator.press(`${OPERATION_MODIFIER}+v`);
+};
+
+test.beforeEach(async ({page, context, browserName}) => {
+  await page.goto(TEST_CONST.LOCAL_URL, {waitUntil: 'load'});
+  if (browserName === 'chromium') await context.grantPermissions(['clipboard-write', 'clipboard-read']);
```

Comment:
Let's use curly braces for conditionals everywhere. I believe we should have `curly` rule enabled everywhere, can you please confirm it?
```suggestion
  if (browserName === 'chromium') {
    await context.grantPermissions(['clipboard-write', 'clipboard-read']);
  }
```

Reply:
it wasn't enabled, i've added it